Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1061,6 +1014,8 @@ namespace Babylon
         arcana::make_task(arcana::threadpool_scheduler, m_cancelSource,
             [this, dataSpan, generateMips, invertY]() {
                 bimg::ImageContainer* image = bimg::imageParse(&m_allocator, dataSpan.data(), dataSpan.size());
+                // todo: bimg::imageParse will return nullptr when trying to load a texture with an url that is not a valid texture
+                // Like a 404 html page.
```

Comment:
that should be okay. it should be caught by the error handler below. is that happening?

Reply:
image is null and get a null pointer exception. FlipY, GenerateMips or CreateTextureFromImage do not provide test for null pointer. What kind of errors are handled? It's not obvious to me

Reply:
arcana tasks will eat all exceptions and hold on to them for the next task. If the task chain has a check for exceptions (like it does here), it should be handled. But that said, we can probably check for nullptr explicitly for when imageParse fails.

Reply:
This is something I'll address shortly as I need to, basically, do the same with empty index/vertex buffer.